tests/testthat/test-plate_builder.R

```diff
@@ -1,3 +1,17 @@
+test_that("Dilution extraction from sample names", {
+  expect_equal(
+    as.vector(sapply(c("POS 1/40", "Unknown4", "CP3 1/50", "P1/2"),
+             extract_dilution_from_name)),
+    c("1/40", NA, "1/50", "1/2")
```

Comment:
`stringr::str_extract(sample_name, dilution_regex)` is already a vectorized operation. There is no need to do sapply